formulaire/src/main/java/fr/openent/formulaire/service/impl/DefaultResponseService.java

```diff
@@ -91,14 +91,15 @@ public void getMissingResponses(String formId, String distributionId, Handler<Ei
 
     @Override
     public void create(JsonObject response, UserInfos user, String questionId, Handler<Either<String, JsonObject>> handler) {
-        String query = "INSERT INTO " + RESPONSE_TABLE + " (question_id, choice_id, answer, responder_id, distribution_id) " +
-                "VALUES (?, ?, ?, ?, ?) RETURNING *;";
+        String query = "INSERT INTO " + RESPONSE_TABLE + " (question_id, choice_id, answer, responder_id, distribution_id, choice_index) " +
+                "VALUES (?, ?, ?, ?, ?, ?) RETURNING *;";
         JsonArray params = new JsonArray()
                 .add(questionId)
                 .add(response.getInteger(CHOICE_ID, null))
                 .add(response.getString(ANSWER, ""))
                 .add(user.getUserId())
-                .add(response.getInteger(DISTRIBUTION_ID, null));
+                .add(response.getInteger(DISTRIBUTION_ID, null))
```

Comment:
idem (cf commentaire niveau vérif)

Reply:
Cf plus haut, le champ SQL à une règle NOT NULL